### DISCUSSION TITLE
Move window to adjacent position

### DISCUSSION THREAD

**Discussion by jay-meister** [rxhanson/RectanglePro-Community#12] [Q&A]
I have set up a simple cycling of 1/4 -> 1/3 -> 1/2 -> 2/3 -> 3/4 for both left and right:

<img width="641" alt="Screenshot 2022-02-02 at 16 08 56" src="https://user-images.githubusercontent.com/12638261/152191713-87a3346a-9d8c-41cd-95d4-a8f9506b328f.png">
 
I want to be able to shift my 1/4 window 100% to the right or left - so question is: how can I configure the 'move window to the adjacent position'

**Comment by rxhanson** [comment]
Currently this requires a "Custom" Docking Position. Here's a quick example:

![image](https://user-images.githubusercontent.com/13651296/152270659-14a0fca0-fb20-43f0-b3ef-acb32f51d848.png)

I do have plans to make this simpler with a concept of a custom shortcut relative to the current size/position, since this can be a bit tedious in its current form.

**Reply by jay-meister** [reply]
Ah ok, I really want that "relative to current size/position" feature then. 🤞 that gets implemented.

Thanks for all the work on Rectangle!

**Reply by jay-meister** [reply]
Should I mark this as answered? and should I open a feature request discussion point?

**Reply by rxhanson** [reply]
Up to you :) It's on my radar, and I don't think having it as a separate discussion will accelerate implementation, but it doesn't bother me if there's a separate discussion around it either.
